src/block_disagg/block_disagg_ckpt.c

```diff
@@ -174,12 +174,44 @@ __wt_block_disagg_checkpoint_resolve(WT_BM *bm, WT_SESSION_IMPL *session, bool f
     } else {
         /* Keep all metadata for regular tables. */
         WT_SAVE_DHANDLE(
-          session, ret = __block_disagg_update_shared_metadata(bm, session, tablename, md_value));
+          session, ret = __block_disagg_update_shared_metadata(bm, session, md_key, md_value));
+        WT_ERR(ret);
+
+        /* Check if we need to include any other metadata keys. */
+        if (WT_SUFFIX_MATCH(block_disagg->name, ".wt")) {
+            /* TODO: Less hacky way of finding related metadata. */
+
+            WT_ERR(__wt_snprintf(md_key, len, "colgroup:%s", block_disagg->name));
```

Comment:
When we create a `table:` object, WT also creates one column group. I don't know what would happen without it, but since WT creates it, we may as well recreate that metadata on the follower.

Reply:
Oh, I see. That makes sense, thanks for explaining it.